course_discovery/apps/api/v1/tests/test_views/test_programs.py

```diff
@@ -298,3 +314,71 @@ def test_list_exclude_utm(self):
     def test_minimal_serializer_use(self):
         """ Verify that the list view uses the minimal serializer. """
         assert ProgramViewSet(action='list').get_serializer_class() == MinimalProgramSerializer
+
+    def test_create_using_api(self):
+        """
+        Verify endpoint successfully creates a program.
+        """
+        response = self.client.post(self.list_path, self._program_data(), format='json')
+        assert response.status_code == 201
+        program = Program.objects.last()
+        assert program.title == response.data['title']
+        assert program.status == response.data['status']
+        assert program.courses.count() == 3
+        assert program.authoring_organizations.count() == 3
```

Comment:
@iam-mhaseeb @ziafazal  In our case there will be only one edx organization which creates and authors its program. How we will ensure this?

Reply:
True, in our case authoring_organizations or credit_backing_organizations would be only one edx organization. As we would be sending our data via server to server communication so we will be sending only one edx organization from wordpress which we can get either from edly-user-info cookie or edx-org set in Edly_settings.